extensions/README.md

```diff
@@ -45,22 +45,22 @@ stable for over a year and are used in twenty or more implementations.
-| [Commons](commons/README.md) (-)                             | Item, Collection | Provides a way to specify data fields in a collection that are common across the STAC Items in that collection, so that each does not need to repeat all the same information. | *Proposal* |
-| [Data Cube](datacube/README.md) (`cube`)                     | Item, Collection | Data Cube related metadata, especially to describe their dimensions. | *Proposal* |
-| [EO](eo/README.md) (`eo`)                                    | Item             | Covers electro-optical data that represents a snapshot of the earth for a single date and time. It could consist of multiple spectral bands, for example visible bands, infrared bands, red edge bands and panchromatic bands. The extension provides common fields like bands, cloud cover, gsd and more. | *Pilot* |
-| [Label](label/README.md) (`label`)                           | Item             | Items that relate labeled AOIs with source imagery | *Proposal* |
-| [Point Cloud](pointcloud/README.md) (`pc`)                   | Item             | Provides a way to describe point cloud datasets. The point clouds can come from either active or passive sensors, and data is frequently acquired using tools such as LiDAR or coincidence-matched imagery. | *Proposal* |
-| [Projection](projection/README.md) (`proj`)                  | Item             | Provides a way to describe items whose assets are in a geospatial projection. | *Proposal* |
-| [SAR](sar/README.md) (`sar`)                                 | Item             | Covers synthetic-aperture radar data that represents a snapshot of the earth for a single date and time. | *Proposal* |
-| [Satellite](sat/README.md) (`sat`)                           | Item             | Satellite related metadata for data collected from satellites. | *Proposal* |
-| [Scientific](scientific/README.md) (`sci`)                   | Item, Collection | Scientific metadata is considered to be data that indicate from which publication data originates and how the data itself should be cited or referenced. | *Proposal* |
-| [Single File STAC](single-file-stac/README.md) (-)           | ItemCollection   | An extension to provide a set of Collections and Items as a single file catalog. | *Proposal* |
-| [Versioning Indicators](version/README.md) (-)               | Item, Collection | Provides fields and link relation types to provide a version and indicate deprecation. | *Proposal* |
-| [View Geometry](view/README.md)                              | Item             | View Geometry adds metadata related to angles of sensors and other radiance angles that affect the view of resulting data | *Proposal* |
+| Extension Title                                | Name             | Prefix   | Scope                     | Description                       | Maturity |
```

Comment:
Should we replace "name" with "identifier"? "name" is so ambiguous. Then we could say in the "stac_extensions" definitions that the identifier needs to be used. And then at the end of this doc where it explains that stac_extension uses the folder names say that the identifier is the folder name.

Reply:
And maybe put the name/identifier and prefix column at the end? From a user perspective seems to be the least interesting information in the overview table.

Reply:
I'm good with all of these changes, but I don't think it's necessary to convey to the users that the directory name is the identifier.  The `identifier` field in the extension's README and the `identifier` in each extension's README should make it crystal clear.

Reply:
It's more as a clarification for extension writers. Not sure whether it's completely clear to "newbies" that the identifier = folder name?! And I'm really just speaking to put it at the end of the file where it already speaks about folder names.

Reply:
I totally agree, but so far the confusion has come from implementors because it's a bit weird to only be able to identify an extension by it's folder name.  It's just yet another nuance -- The directory name should match the identifier, but the instructions to consumers/users shouldn't be to go look at an extension's path to find the directory name, it should just be something that is defined in the extension itself (which it is with this PR).  So I'm just modifying the text to state that extension creators should name the directory with the extension's identifier.

Reply:
How stuck are you on these fields being at the end of the table?  It seems most tables like this in tech documentation will have all of the various identifiers, unique values, etc at the beginning and longer text descriptions as the last column.

I don't know if we want to make this decision based on the neatness of the raw markdown, but due to the varying links of each entry's description, adding fields after that makes the markdown very nasty.  

It may look cleaner in both the raw and rendered markdown if we instead move the maturity field ahead of the description and leave the description as the last column?  Thoughts?

Reply:
Agreed.

Not so stuck on the order. I just looks a bit messy on the rendered GitHub html pages and repeats quite a lot (EO, eo, eo).